test/parallel/test-crypto-hash.js

```diff
@@ -154,3 +154,10 @@ common.expectsError(
     message: 'The "algorithm" argument must be of type string'
   }
 );
+
+{
+  const Hash = crypto.Hash;
+  const instance = crypto.Hash('sha256');
+  assert(instance instanceof Hash, 'Hash is expected to return a new instance' +
```

Comment:
needs a space either at the end of this line (after instance) or on the next line. Right now it says "instancewhen".

Reply:
Ah, Sorry. I'll update it soon.